docs/src/main/sphinx/connector/hive.rst

```diff
@@ -1062,13 +1062,14 @@ format, which has the schema set based on an Avro schema file/literal. Trino is
 also capable of creating the tables in Trino by infering the schema from a
 valid Avro schema file located locally, or remotely in HDFS/Web server.
 
-To specify that the Avro schema should be used for interpreting table's data one must use ``avro_schema_url`` table property.
-The schema can be placed remotely in
-HDFS (e.g. ``avro_schema_url = 'hdfs://user/avro/schema/avro_data.avsc'``),
-S3 (e.g. ``avro_schema_url = 's3n:///schema_bucket/schema/avro_data.avsc'``),
-a web server (e.g. ``avro_schema_url = 'http://example.org/schema/avro_data.avsc'``)
-as well as local file system. This URL, where the schema is located, must be accessible from the
-Hive metastore and Trino coordinator/worker nodes.
+To specify that the Avro schema should be used for interpreting table data, use
+the ``avro_schema_url`` or the ``avro.schema.literal`` table property. The
```

Comment:
I think this is wrong

 ``avro.schema.literal`` table property 

I think this is a catalog configuration property

and the _ version is either a table property or a session property

Reply:
``avro.schema.literal`` -> ``avro_schema_literal``

However this paragraph is mostly about `avro_schema_url` and how to eventually retrieve the avro schema of the table.

Reply:
@mosabua  `avro.schema.literal` and `avro.schema.url` are Hive specific table properties.